app/views/conference/schedule.html.haml

```diff
@@ -0,0 +1,81 @@
+.container
+  = render partial: 'schedule_tabs',  locals: { active: 'schedule' }
+
+  #schedule-content
+    %h1.text-center
+      Schedule for
+      = @conference.title
+    .dropdown.schedule-dropdown
+      %button{ type: "button", class: "btn btn-default dropdown-toggle", 'data-toggle' => "dropdown" }
+        = @day
+        %span.caret
+      %ul.dropdown-menu
+        - @dates.each do |date|
+          %li.li-dropdown-schedule{ class: "#{ 'active' if @day == date }" }
+            = link_to date, "#" + "#{date}", "data-toggle" => "tab", "class" => "date-tab"
+
+    .tab-content
+      - @dates.each do |date|
+        %div{ class: "tab-pane #{ 'active' if @day == date }", id: "#{ date }" }
+
+          .visible-xs-inline
+            = render partial: 'carousel', locals: { date: date, number_columns: 1 }
+
+          .visible-sm-inline
+            = render partial: 'carousel', locals: { date: date, number_columns: 2 }
+
+          .visible-md-inline.visible-lg-inline
+            = render partial: 'carousel', locals: { date: date, number_columns: 3 }
+
+:javascript
+  // change of active tab and the button title when a date is clicked
+  $(function() {
+    $('.date-tab').on('click', function(e) {
```

Comment:
But that is for nav tabs and I am using a dropdown, so I think I can not use it :confused: